### ISSUE TITLE
[🐛]: Create/GT Pipe/Pump Integration Issues

### ISSUE THREAD

**Issue by Supermarcel10** [TerraFirmaGreg-Team/Modpack-Modern#581]
### Version modpack Used

0.7.14

### Environment

Forge Server & Singleplayer

### New Worlds

Yes

### Expected Behavior

Create pump/pipe works straight away and detects a GT fluid pipe has been attached.

![image](https://github.com/user-attachments/assets/a968d3b3-0a04-4235-8ffa-824650509913)

### Actual Behavior

Create pumps/pipes do not seem to integrate/update when GT fluid pipes are used. They only update once they are rotated, and realise a GT fluid pipe is attached to them.

![image](https://github.com/user-attachments/assets/629505b4-e52c-480a-ac6c-9a94468955d0)

### Reproduce
1) Get a Create pump and any type of GT fluid pipe(s).
2) Place and power the pump.
3) Connect pump with any type of fluid pipes to a source.

![image](https://github.com/user-attachments/assets/066f701e-8dac-471f-acfe-becc6fd7aa47)

4) Place the GT fluid pipes (as pictured as an example).

![image](https://github.com/user-attachments/assets/629505b4-e52c-480a-ac6c-9a94468955d0)

The fluid will not flow.

### Temporary solution
Rotate the pump twice.

![image](https://github.com/user-attachments/assets/25a96a76-c69f-4357-8790-fb84bd1cf3aa)
![image](https://github.com/user-attachments/assets/a968d3b3-0a04-4235-8ffa-824650509913)

### Additional Information

This can likely be fixed by updating GregTech and/or Create, or referencing this issue to them as an issue with integrating with the other mod.

**Comment by Mqrius** [comment]
Can't reproduce, can anyone doublecheck if this is still an issue? However I try it it seems to just connect fine. Might've been fixed in a random update some time.

(I did spot a minor texturing issue though where the connected side of the huge pipe is transparent)

<img width="1805" height="1008" alt="Image" src="https://github.com/user-attachments/assets/2b086d89-457b-4842-a866-2155aa148b9d" />

**Comment by Pyritie** [comment]
true, I did notice we haven't gotten any "why is my pipe not working?" questions recently
